### DISCUSSION TITLE
Payment for one or several projects

### DISCUSSION THREAD

**Discussion by moviex1** [Laravel-Backpack/community-forum#1127] [Q&A (Help)] [answered]
if i have prod and dev servers, can i use single project subscription for both?

**Comment by karandatwani92** [comment, marked as answer]
Hey @moviex1

Here's a complete answer to your [question](https://backpackforlaravel.com/docs/6.x/faq#do-i-need-a-license-to-put-a-pro-project-on-a-testing-domain).

**Reply by moviex1** [reply]
thank you so much!

**Reply by karandatwani92** [reply]
If you like Backpack so far, please give it a star on the [Github repo](https://github.com/Laravel-Backpack/CRUD). Thanks
